libdnf/rpm/solv/map.cpp

```diff
@@ -0,0 +1,127 @@
+/*
```

Comment:
I'm pretty sure this commit could use some reasoning behind it broken down in the commit message :slightly_smiling_face: 

Also, as a side, note, it seems we are slowly creating basically a C++ wrapper around libsolv here. Can't say I disagree with that :grin: